src/azure-cli/azure/cli/command_modules/appservice/appservice_environment.py

```diff
@@ -118,7 +122,7 @@ def _get_ase_client_factory(cli_ctx, api_version=None):
 
 def _get_deployment_client_factory(cli_ctx):
     from azure.mgmt.resource import ResourceManagementClient
-    client = get_mgmt_service_client(cli_ctx, ResourceManagementClient).deployments
+    client = get_mgmt_service_client(cli_ctx, ResourceManagementClient, api_version=VERSION_2019_10_01).deployments
```

Comment:
Is it feasible to use a central configuration center instead of specifying in each method? E.g. _shared.py

Reply:
Great suggestion, I have centralized the api_version, but made it possible to override in a specific method if needed (e.g. to use new features). I also made a few other optimizations now that the file was open 👍